velox/expression/CastExpr-inl.h

```diff
@@ -275,6 +275,28 @@ VectorPtr CastExpr::applyDecimalToIntegralCast(
   return result;
 }
 
+template <typename FromNativeType, TypeKind ToKind>
+VectorPtr CastExpr::applyDecimalToBooleanCast(
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context,
+    const TypePtr& fromType,
+    const TypePtr& toType) {
+  using To = typename TypeTraits<ToKind>::NativeType;
+  VectorPtr result;
+  context.ensureWritable(rows, toType, result);
+  (*result).clearNulls(rows);
+  auto resultBuffer = result->asUnchecked<FlatVector<To>>()->mutableRawValues();
```

Comment:
// Bool uses compact representation, use mutableRawValues<uint64_t> and
  // bits::setBit instead.
  T* mutableRawValues() {

Reply:
Thanks, fixed it.